### ISSUE TITLE
POD improvement

### ISSUE THREAD

**Issue by guimard** [lstein/Perl-GD#29]
man reports an error to split this. Little patch:
```
--- a/lib/GD.pm
+++ b/lib/GD.pm
@@ -1305,8 +1305,13 @@

 Creates a new palette image from a truecolor image.

-  samplefactor The quantization precision between 1 (highest quality) and 10 (fastest).
-  maxcolor     The number of desired palette entries.
+=over 4
+
+=item samplefactor: The quantization precision between 1 (highest quality) and 10 (fastest).
+
+=item maxcolor: The number of desired palette entries.
+
+=back

 This is the same as createPaletteFromTrueColor with the
 quantization method GD_QUANT_NEUQUANT. This does not support dithering.
```

**Comment by guimard** [comment]
Thanks !

**Comment by rurban** [comment]
Released with 2.71
